### DISCUSSION TITLE
mrisoc is a p.o.s. and should be banned from this group!

### DISCUSSION THREAD

**Discussion by ledcomp** [Jyers/Marlin#1916] [General Discussion]
mrisoc is a p.o.s. and should be banned from this group! I am tired of his bans and general BS. Keep him out of here. Now the asshat completely blocked me from r/ender3v2. When is he going to get off of his power trip?

**Comment by CRCinAU** [comment]
Please do not contribute to the already toxic nature of this matter.

This isn't an 'us vs them' situation.
